webui.py

```
@@ -158,6 +170,9 @@ async def run_browser_agent(
                 use_vision=use_vision,
                 max_actions_per_step=max_actions_per_step,
                 tool_calling_method=tool_calling_method
+                gbc=gbc,
```

Ruff (0.8.2)

SyntaxError: Expected ',', found name

---

Trailing comma missing

(COM812)